controllers/baremetalset_controller.go

```diff
@@ -477,9 +477,8 @@ func (r *BaremetalSetReconciler) baremetalHostProvision(instance *ospdirectorv1b
 	templateParameters = make(map[string]interface{})
 	templateParameters["MgmtIp"] = ip.String()
 	templateParameters["MgmtInterface"] = instance.Spec.MgmtInterface
-	templateParameters["MgmtGateway"] = "192.168.25.1" //FIXME
+	templateParameters["MgmtGateway"] = ipset.Status.Networks["ctlplane"].Gateway
```

Comment:
right now this is not really Mgmt, this is Ctl, should we rename all of this and follow up with adding Mgmt net implementation?

Reply:
Yes. We had agreed to do this just got forgotted I think. Anyways I filed this: https://github.com/openstack-k8s-operators/osp-director-operator/issues/92